docs/local_setup/quick_setup.MD

```diff
@@ -10,7 +10,7 @@ sorry _Windows_ users.
 
 How to start on your local machine
 1. ./vendor/bin/sail up
-2. execute **php artisan migrate:refresh --seed**
+2. execute **php artisan migrate:refresh --seed** in the _containerized environment_
```

Comment:
@dembygenesis take a look at the changes that I did.

Reply:
Okay, I have taken a look at this comment - and it is a good addition/revision.